benchmarks/c/miscellaneous/jumpIntoLoop.c

```diff
@@ -0,0 +1,19 @@
+#include <stdint.h>
+#include <assert.h>
+#include <dat3m.h>
+
+volatile int32_t x = 0;
+
+int main()
+{
+    int i = __VERIFIER_nondet_int();
+    if (i == 42) goto L;
+
+    for (i = 0; i < 5; i++) {
+L:
+    x++;
+    }
+
+    assert(x <= 5);
```

Comment:
What about this one?
```
#include <stdint.h>
#include <assert.h>
#include <dat3m.h>

volatile int32_t x = 0;

int main()
{
    int j = 0;
    int i = __VERIFIER_nondet_int();
    if (i == 42) goto L;

    for (j = 1; j < 5; j++) {
L:
        x++;
    }

    assert((i != 42 || x == 5) || (i == 42 || x == 5));
    return 0;
}
```

Reply:
How is this easier to read then `(jumpIntoLoop => x == 5) && (!jumpIntoLoop => x == 4)`? You can also abbreviate the variable to `b` if you want it short. Essentially, you just replace the arbitrary `i == 42` with a less arbitrary `b`, since the value of `i` does not matter in the test anymore.

Reply:
Ok, now I understand what you meant.